returnn/config.py

```diff
@@ -78,6 +78,7 @@ def __init__(
         python_epilog="",
         python_epilog_hash=None,
         hash_full_python_code=False,
+        sort_dicts=False,
```

Comment:
Default value should be old behaviour i.e. sorting enabled?
```suggestion
        sort_dicts=True,
```